# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515978812 28800
#      Sun Jan 14 17:13:32 2018 -0800
# Branch stable
# Node ID 49876d80d110f303562f8d9ae5014ef758c4e3f6
# Parent  5ed2e9248ee98a78a991abe4e96a2c2d22cb426c
doc: Remove backslashes before double quotes in m-file docstrings (bug #52870).
Documentation converted from C++ had backslashes before double quotes.

* fnmatch.m, md5sum.m, usage.m: Remove backslashes before double quotes.

diff --git a/scripts/deprecated/fnmatch.m b/scripts/deprecated/fnmatch.m
--- a/scripts/deprecated/fnmatch.m
+++ b/scripts/deprecated/fnmatch.m
@@ -23,17 +23,17 @@
 ## Please use @code{glob} or @code{regexp} in all new code.
 ##
 ## Return true or false for each element of @var{string} that matches any of
 ## the elements of the string array @var{pattern}, using the rules of
 ## filename pattern matching.  For example:
 ##
 ## @example
 ## @group
-## fnmatch (\"a*b\", @{\"ab\"; \"axyzb\"; \"xyzab\"@})
+## fnmatch ("a*b", @{"ab"; "axyzb"; "xyzab"@})
 ##      @result{} [ 1; 1; 0 ]
 ## @end group
 ## @end example
 ## @seealso{glob, regexp}
 ## @end deftypefn
 
 ## Deprecated in version 4.0
 
diff --git a/scripts/deprecated/md5sum.m b/scripts/deprecated/md5sum.m
--- a/scripts/deprecated/md5sum.m
+++ b/scripts/deprecated/md5sum.m
@@ -20,23 +20,23 @@
 ## @deftypefn  {} {} md5sum (@var{file})
 ## @deftypefnx {} {} md5sum (@var{str}, @var{opt})
 ##
 ## @code{md5sum} is deprecated and will be removed in Octave version 4.6.
 ## For equivalent functionality replace calls like @code{md5sum (@var{file})}
 ## with:
 ##
 ## @example
-## hash (\"md5\", fileread (@var{file}))
+## hash ("md5", fileread (@var{file}))
 ## @end example
 ##
 ## And calls like @code{md5sum (@var{str}, true)} with:
 ##
 ## @example
-## hash (\"md5\", fileread (@var{str}))
+## hash ("md5", fileread (@var{str}))
 ## @end example
 ##
 ## Calculate the MD5 sum of the file @var{file}.
 ##
 ## If the second parameter @var{opt} exists and is true, then calculate the MD5
 ## sum of the string @var{str}.
 ##
 ## @seealso{hash, fileread}
diff --git a/scripts/deprecated/usage.m b/scripts/deprecated/usage.m
--- a/scripts/deprecated/usage.m
+++ b/scripts/deprecated/usage.m
@@ -34,17 +34,17 @@
 ## from an improper call to a function, such as calling a function with an
 ## incorrect number of arguments, or with arguments of the wrong type.  For
 ## example, most functions distributed with Octave begin with code like
 ## this
 ##
 ## @example
 ## @group
 ## if (nargin != 2)
-##   usage (\"foo (a, b)\");
+##   usage ("foo (a, b)");
 ## endif
 ## @end group
 ## @end example
 ##
 ## @noindent
 ## to check for the proper number of arguments.
 ## @seealso{print_usage}
 ## @end deftypefn
